spec/models/user_spec.rb

```diff
@@ -51,7 +52,7 @@
   end
 
   describe "supervisors" do
-    context "#volunteers_serving_transistion_aged_youth" do
+    describe "#volunteers_serving_transistion_aged_youth" do
```

Comment:
Yeah, can you clarify a bit more on why you changed some of these? What was your standard for `describe` over `context` and vice versa?